css/style.css

```diff
@@ -0,0 +1,1140 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #fff;
+  overflow-x: hidden;
+  font-family: 'Inter', sans-serif;
+  font-style: none;
+}
+
+.header-container {
+  width: 375px;
+  height: 812px;
+  background: #fff;
+}
+
+.nav-top {
+  position: fixed;
+  margin: 15px 0;
+  display: flex;
+  flex-direction: row;
+  padding: 0 35px;
+  align-items: center;
+  width: 375px;
```

Comment:
- [ ] Kindly make sure the entire content of the page fits in without having to scroll. You may consider using the relative units on your width (`%, vw`) instead of `px`
- [ ] Also make sure you stick to the design as much as you can. The typos, spacing, and `color` are supposed to match the design in Figma.
`Hint: Here are the colors:`
![image](https://user-images.githubusercontent.com/97100091/212133600-6a7e84c6-65fa-4b33-b2dd-29f0fbe55859.png)


Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212133037-8bdc7997-3701-4cf6-9a9b-1736e13f0912.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212133125-d1425d26-9c56-4bb8-9566-e0490f21c9fa.png)